### PR TITLE
[WIP ] updating healthcare to health-care

### PR DESCRIPTION
**Pull request by rroueche**

### CODE REVIEW

File: src/js/hca/routes.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { createRoutes as createFormRoutes } from '../common/schemaform/helpers';
 import formConfig from './config/form';
-import HealthCareApp from './HealthCareApp.jsx';
+import HealthCareApp from './health-careApp.jsx';
```
Comment (jbalboni) [inline src/js/hca/routes.jsx]: @rroueche This should be changed back.

**Comment by jbalboni** [comment]
@rroueche HCA looks ok once you make the change I mentioned. I don't know exactly how the healthcare -> health-care redirects work with it, though. It would be good to test that somewhere (maybe we have to wait until staging?).

File: src/js/hca/routes.jsx (hunk 1 above)
Comment (rroueche) [inline src/js/hca/routes.jsx]: Ok - that has been reverted @jbalboni !

Comment (jbalboni) [inline src/js/hca/routes.jsx]: @rroueche Needs to be capitalized still, too :)

**Comment by U-DON** [comment]
Heads up, #5770 adds some links that will use `healthcare` in the URL until this rolls out, so note that should be changed here when #5770 gets merged to master and master gets merged into this branch.

**Comment by patrickvinograd** [comment]
Closing in lieu of #5864

**Pull request closed without merging**